sqlglot/dialects/tsql.py

```diff
@@ -283,6 +284,8 @@ class Tokenizer(tokens.Tokenizer):
 
         KEYWORDS = {
             **tokens.Tokenizer.KEYWORDS,
+            "BEGIN": TokenType.COMMAND,
+            "SET": TokenType.SET,
```

Comment:
This and `"WITH": TokenType.WITH` are redundant, these mappings are already in the base tokenizer.

Reply:
Removed.